arch/x86/ept.c

```diff
@@ -83,7 +83,7 @@ static void *find_next_table(uint32_t table_offset,
 	}
 
 	/* Get address of the sub-table */
-	sub_table_addr = (void *)(table_entry & IA32E_REF_MASK);
+	sub_table_addr = (void *)(table_entry & CPU_PAGE_MASK);
```

Comment:
This is for the EPT usage.
It will be better that the IA32E_REF_MASK is still used.(The bit63 should be cleared).